### ISSUE TITLE
Feedback on Single Logout Wanted

### ISSUE THREAD

**Issue by AndersAbel** [KentorIT/authservices#411]
Earlier today I published version 0.17.0 which includes single logout support. It's a major piece of functionality where a number of different scenarios need to be supported.

Now I want feedback!
* If you get it working, please leave a comment below.
* If you run into problems, please open an issue for that. Then leave a comment here with the issue number.

**Comment by AndersAbel** [comment]
Ping @henningjensen, @cederlof.

**Comment by AndersAbel** [comment]
ping @johansvard

**Comment by federicobarera** [comment]
Here : https://github.com/KentorIT/authservices/issues/421 :)

**Comment by dmeierotto** [comment]
Here is another issue, related to a shibboleth idp: [#435](https://github.com/KentorIT/authservices/issues/425)

**Comment by AndersAbel** [comment]
@dmeierotto Thanks. I've commented on that issue on what needs to be done.

**Comment by federicobarera** [comment]
https://github.com/KentorIT/authservices/issues/441

**Comment by dmeierotto** [comment]
Please see #448

**Comment by henningjensen** [comment]
Works!

**Comment by fruitraccoon** [comment]
I've got it working too - thanks very much!
While it isn't affecting us, I did notice something: it appears that SP-initiated logouts always use the `Redirect` binding, even if the `IdentityProvider.SingleLogoutServiceBinding` is set to `Post` - see [here](https://github.com/KentorIT/authservices/blob/455a635397edad80202d4b9a1aa2b7f12d550f93/Kentor.AuthServices/WebSSO/LogOutCommand.cs#L137).  Perhaps it should be using the SingleLogoutServiceBinding value, like [here](https://github.com/KentorIT/authservices/blob/455a635397edad80202d4b9a1aa2b7f12d550f93/Kentor.AuthServices/WebSSO/LogOutCommand.cs#L199)?
Not familiar enough with the code/SAML spec at the moment to know if this is by-design :)
